.github/template-files/config.yml

```diff
@@ -0,0 +1,50 @@
+conda/governance:
+  # [required] community files
+  - CODE_OF_CONDUCT.md
```

Comment:
This is all thats needed to resolve https://github.com/conda-incubator/conda-recipe-manager/issues/145